datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -178,12 +178,15 @@ def update_check(self):
         command = ['docker', 'exec', self.container_name]
         command.extend(get_pip_exe(self.python_version))
         command.extend(('install', '-e', self.check_mount_dir))
+        if file_exists(path_join(get_root(), self.check, 'requirements.in')):
+            command.extend(('-r', path_join(self.check_mount_dir, 'requirements.in')))
```

Comment:
(nit) Could you use a constant for `requirements.in` ?

Reply:
Done.

Reply:
Thx 👍